packages/sitecore-jss/src/layout-service.ts

```diff
@@ -58,6 +59,17 @@ export type RestLayoutServiceConfig = {
   dataFetcherResolver?: DataFetcherResolver;
 };
 
+export type GraphQLLayoutServiceConfig = {
+  /**
+   * Your Graphql endpoint
+   */
+  apiHost: string;
```

Comment:
We use "apiHost" here and "endpoint" for our `GraphQLSitemapService`. We should probably be consistent among these (and the upcoming `GraphQLDictionaryService`).

Reply:
Agree, changed